src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -117,6 +118,12 @@ public void parseCommand_select() throws Exception {
     }
 
     @Test
+    public void parseCommand_sort() throws Exception {
+        assertTrue(parser.parseCommand(SortCommand.COMMAND_WORD) instanceof SortCommand);
+        assertTrue(parser.parseCommand(SortCommand.COMMAND_WORD + " 3") instanceof SortCommand);
```

Comment:
Why should this work? What's the extra argument for?

Reply:
The help and list parser did the same thing, I thought it would be more coherent if i stick to the original style

Reply:
Okay noted!